recipes/LibriSpeech/ASR/CTC/hparams/downsampled/train_hf_wavlm_signal_downsampling.yaml

```diff
@@ -44,7 +44,6 @@ sample_rate: 16000
 downsampling_factor: 3
 upsampling: True
 use_language_modelling: True
```

Comment:
if this is True then we must have `ngram_lm_path` see https://github.com/speechbrain/speechbrain/blob/d84d21b8cb245ed6de509b58bad90e9283cefbab/recipes/LibriSpeech/ASR/CTC/train_with_wav2vec.py#L393  (it applies for the others too)

Reply:
So for that variable there is a bug in the test regex that misses the variable. I'll check if that is an easy fix.